packages/ses/src/lockdown-shim.js

```diff
@@ -317,6 +296,9 @@ export const repairIntrinsics = (options = {}) => {
 
   /**
    * 3. HARDEN to share the intrinsics.
+   *
+   * We define hardenIntrinsics here so that options are in scope, but return
+   * it to the caller so we can benchmark repair separately from hardening.
```

Comment:
The other motivation is to prepare for vetted shims, which would run after repair and before hardening. In fact, this was the original and primary motivation.